src/background/install-data-generator.ts

```diff
@@ -40,7 +40,7 @@ export class InstallDataGenerator {
             year: currentDate.getUTCFullYear(),
         };
 
-        this.storageAdapter.setUserData({ [LocalStorageDataKeys.installationData]: this.installationData });
+        this.storageAdapter.setUserData({ [LocalStorageDataKeys.installationData]: this.installationData }).catch(console.log);
```

Comment:
If you use console.error instead, the e2e tests should pick it up as a test failure should it occur

Reply:
oh... that's a good point!